app/controllers/courses_controller.rb

```diff
@@ -269,6 +271,29 @@ def update_courses_wikis
     @course.update_wikis(new_wikis)
   end
 
+  def courses_namespaces_params
+    params
+      .require(:course)
+      .permit(namespaces: [wiki: [:language, :project], namespaces: []])
+  end
+
+  def update_courses_namespaces
+    wiki_ns_obj = courses_namespaces_params[:namespaces]
+    wiki_ns_obj.each do |obj|
```

Comment:
What kind of object is this? There's almost always a better variable name than `obj`.